### DISCUSSION TITLE
I'm just too stupid

### DISCUSSION THREAD

**Discussion by Pixsy1929** [typekcz/beepboop-steam#17] [General]
I'm just too stupid 
if anyone could make video about it to how I should use this I would appreciate that

**Comment by typekcz** [comment]
Hello, I can give you a hand.

You need [Docker](https://www.docker.com/) to run this.
Then you can download this repository and unpack it somewhere if you downloaded zip.
Before running it you need to supply configuration files ".env" and "config.json" to project folder, file contents are described [here](https://github.com/typekcz/beepboop-steam#configuration).
Now everything should be ready and you can run it by opening terminal/cmd/powershell in the project folder and running: "docker-compose up -d".
One issue can arise: Steam Guard. In this case you need [VNC client](https://www.realvnc.com/en/connect/download/viewer/). You use it to connect to localhost:5900 there you will have to fill in Steam Guard code and log in (this should be one time thing, you can tick remmember and never worry about it again).

That should be all. Running this project without some basic development knowledge is hard, it's meant to be run on a server.
Let me know if this provides some help and also some background about your use and your level of "computer knowledge" so I can help you better.
